### DISCUSSION TITLE
Integrating kube-bench with AWS Security Hub and editing cfg/eks-1.0/config.yaml

### DISCUSSION THREAD

**Discussion by ismailyenigul** [aquasecurity/kube-bench#799] [Development]
I would like to integrate kube-bench with AWS Security Hub
According to 
https://github.com/aquasecurity/kube-bench/blob/main/docs/asff.md#configure-and-rebuild-kube-bench

I have to edit `cfg/eks-1.0/config.yaml` to specify the AWS account, AWS region, and the EKS Cluster ARN.

Do I have to really edit this file to set these env before building docker image for EKS?

Does it work if I set these env with k8s configmap or env variable instead of hard-coded variables in config file?

Thanks

**Comment by lizrice** [comment]
This is now covered by @mkandelaars's contribution, as discussed in #791 - you can now set these values in a ConfigMap
